pyproject.toml

```diff
@@ -58,7 +58,7 @@ ruff = "ruff check ."
 mypy = "mypy -p pygls"
 check_generated_client = "python scripts/check_client_is_uptodate.py"
 check_commit_style = "npx commitlint --from origin/main --to HEAD --verbose --config commitlintrc.yaml"
-generate_client = "python scripts/generate_client.py --output pygls/lsp/client.py"
+generate_client = "python scripts/generate_client_server.py pygls/lsp"
```

Comment:
Should we rename the command to `generate_client_server`?

Reply:
I've renamed it and the script to `generate_code` - seems to complement the `check_generated_code_is_uptodate.py` script?